content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -4,14 +4,83 @@ page_title: "Teams - Terraform Enterprise"
+
+Teams can only have permissions on workspaces within their organization, although any user in the team can belong to teams in other organizations.
+
+If a user belongs to at least one team in an organization, they are considered a member of that organization.
+
+## The Owners Team
+
+Every organization has a team named `owners`, whose members have special permissions. In TFE's documentation and UI, members of the owners team are sometimes called organization owners.
+
+An organization's creator is the first member of its owners team; other members can be added or removed in the same way as other teams. Unlike other teams, the owners team can't be deleted and can't be empty; if there is only one member, you must add another before removing them.
+
+Members of the owners team have [admin permissions](./permissions.html#admin) on every workspace in the organization. Additionally, the following tasks can only be performed by organization owners:
+
+- Creating new workspaces
+- Managing Sentinel policies
+- Overriding `soft mandatory` Sentinel policies
+- Creating and deleting teams
+- Managing team membership
+- Viewing the full list of teams, including teams the user does not belong to
+- Managing [organization settings][]
+
+## Managing Teams
+
+Teams are managed in the [organization settings][]. Click the "Settings" link in the top navigation bar, then click the "Teams" link in the left sidebar.
+
+![Screenshot: the teams page, displaying a list of teams. Each team's entry shows how many members it has.](./images/teams-list.png)
+
+The teams page includes a list of the organization's teams. Clicking a team's name in the list loads its team settings page, where you can manage its membership and settings:
+
+![Screenshot: a team's settings page](./images/teams-team-settings.png)
+
+The team settings page includes a list of the team's current members, with badges to indicate which users have [two-factor authentication](./2fa.html) enabled.
+
+Only organization owners can manage teams; other users can view the teams page in read-only mode. When a non-owner views the teams page, it only includes the current user's teams.
+
+### Creating and Deleting Teams
+
+-> **API:** See the [Teams API](../api/teams.html).
+
+Organization owners can create new teams from the teams page, using the controls under the "Create a New Team" header.
+
+To create a new team, provide a name (unique within the organization) and click the "Create team" button. Team names can include numbers, letters, underscores (`_`), and hyphens (`-`).
+
+To delete a team, go to the target team's settings page and click the "Delete TEAM NAME" button.
+
+### Managing Team Membership
+
+-> **API:** See the [Team Members API](../api/team-members.html).
+
+Organization owners can use a team's settings page to add and remove users from the team.
+
+The structure of your TFE teams should resemble your company's group structure.
```

Comment:
This seems very prescriptive, do we need it? I think if I was going to introduce this at all, it would be "Typically, teams will mirror your company's group structure. For example..." 

but we don't really get questions about this, in practice. The only questions we get are when permission levels don't meet people's needs.

Reply:
One of the few holdovers from the previous iteration of these pages, and it came in when everyone involved was neck-deep in the "recommended practices guide" and very much in a "we must explain to people how to think about this stuff" mode. 

Eh, sure, may as well soften it up a bit. I can just link out to the reccos guide, too.